yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Declination coordinate of the position of the diaObject at time radecTai
   - name: diaObjectId
     "@id": "#DiaObject.diaObjectId"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Unique id.
   - name: gPSFluxChi2
     "@id": "#DiaObject.gPSFluxChi2"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Chi^2 statistic for the scatter of gPSFlux around gPSFluxMean
   - name: gPSFluxErrMean
     "@id": "#DiaObject.gPSFluxErrMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Mean of the diaSource PSF flux errors
   - name: gPSFluxLinearIntercept
     "@id": "#DiaObject.gPSFluxLinearIntercept"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: y-intercept of a linear model fit to diaSource PSF flux vs time
```

Comment:
I think it's probably fine to not have the band mentioned in these kinds of repeated comments, since it's in the field name, but the band name does show up in the descriptions of e.g. gPSFluxSigma and gPSFluxChi2, so we're not fully consistent.